src/main/java/com/linecorp/armeria/client/routing/StaticServerGroupBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.routing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class StaticServerGroupBuilder {
+    private List<DefaultWeightedNode> nodes = new ArrayList<>();
+
+    /**
+     * Add a host to this server group.
+     */
+    public StaticServerGroupBuilder addHost(String name, int port) {
```

Comment:
Given that this is a builder class, I think it's just fine to use the name `host()`